Mage.Client/src/main/java/mage/client/cards/DragCardGrid.java

```diff
@@ -1478,6 +1497,84 @@ public void analyseDeck() {
         MageFrame.getInstance().showMessage(finalInfo);
     }
 
+    public void blingDeck() {
+        if (!(this.mode == Constants.DeckEditorMode.FREE_BUILDING)) {
+            return;
+        }
+
+        if (JOptionPane.showConfirmDialog(null, "Are you sure you want to bling your deck?  This process will add cards!", "WARNING",
+                JOptionPane.YES_NO_OPTION) == JOptionPane.NO_OPTION) {
+            return;
+        }
+
+        HashMap<String, Integer> pimpedSets = new HashMap<>();
+        HashMap<CardView, Integer> pimpedCards = new HashMap<>();
+        pimpedSets.put("CP", 1);
+        pimpedSets.put("JR", 1);
+        pimpedSets.put("MPS", 1);
+        pimpedSets.put("CLASH", 1);
+        pimpedSets.put("ARENA", 1);
+        pimpedSets.put("UGIN", 1);
+        pimpedSets.put("WMCQ", 1);
+        pimpedSets.put("APAC", 1);
+        pimpedSets.put("EURO", 1);
+        pimpedSets.put("FNMP", 1);
+        pimpedSets.put("MGDC", 1);
+        pimpedSets.put("MPRP", 1);
+        pimpedSets.put("EXP", 1);
+        pimpedSets.put("GPX", 1);
+        pimpedSets.put("GRC", 1);
+        pimpedSets.put("MBP", 1);
+        pimpedSets.put("MLP", 1);
+        pimpedSets.put("PLS", 1);
+        pimpedSets.put("PTC", 1);
+        pimpedSets.put("SUS", 1);
+
+        String[] sets = pimpedSets.keySet().toArray(new String[pimpedSets.keySet().size()]);
+        Boolean didModify = false;
+
+        for (ArrayList<ArrayList<CardView>> gridRow : cardGrid) {
+            for (ArrayList<CardView> stack : gridRow) {
+                for (CardView card : stack) {
+                    if (card.getSuperTypes().contains("Basic")) {
```

Comment:
It's kind of the structure that the revamped deck editor comes in.  It's all local so no extra burden on the server.